Libraries/LibWeb/Painting/DisplayListPlayerSkia.cpp

```diff
@@ -651,18 +651,48 @@ void DisplayListPlayerSkia::fill_path_using_paint_style(FillPathUsingPaintStyle
     surface().canvas().drawPath(path, paint);
 }
 
+static SkPaint::Cap to_skia_cap(Gfx::Path::CapStyle const& cap_style)
+{
+    switch (cap_style) {
+    case Gfx::Path::CapStyle::Butt:
+        return SkPaint::kButt_Cap;
+    case Gfx::Path::CapStyle::Round:
+        return SkPaint::kRound_Cap;
+    case Gfx::Path::CapStyle::Square:
+        return SkPaint::kSquare_Cap;
+    default:
```

Comment:
not: I'd probably remove the default: branch and put VERIFY_NOT_REACHED() at the end instead. (Also below)